packages/@lwc/scoped-registry/README.md

```diff
@@ -0,0 +1,138 @@
+# @lwc/scoped-registry
```

Comment:
FYI @jdalton @rwaldron, this is where I describe the API surface I intend to expose for scoped registries. Let me know if it makes sense.